pwa/src/components/DiagnosisForm.test.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react'
+import { render, fireEvent } from '@testing-library/react'
+import DiagnosisForm from './DiagnosisForm'
+
+/*
+
+Diagnosis Fields are stored in DiagnosisFields.json in the format:
+
+{
+    "diagnoses" : [
+        {
+            "name": "Name",
+            "id": "id",
+            "text-input-title": "Title",
+            "text-input-type": "type" (i.e. number)
+            "group-title": "Group Title",
+            "group": [
+                { "name": "Name1", "id": "id1" },
+                { "name": "Name2", "id": "id2" }
+            ]
+        },
+        {...}
+    ]
+}
+*/
+
+import * as data from './DiagnosisFields.json'
```

Comment:
Keep imports together.

Reply:
Resolved in (this commit)[https://github.com/Haiti-Distributed-Healthcare-System/hdhs/pull/84/commits/3197b229929bb40c992e71084d6e3cd667482fba].